cedar-policy-core/src/ast/entity.rs

```diff
@@ -318,6 +318,49 @@ pub struct Entity {
 }
 
 impl Entity {
+    /// The implementation of [`Eq`] and [`PartialEq`] for
```

Comment:
Can you add the feature flag to core as well?

Reply:
Or generally, do you need changes to core at all? It seems you can put everything into the validator.